src/Subtitles/STS.cpp

```diff
@@ -487,6 +500,8 @@ static CStringW WebVTT2SSA(CStringW str)
         str.Replace(L"</b>", L"{\\b}");
         str.Replace(L"<u>", L"{\\u1}");
         str.Replace(L"</u>", L"{\\u}");
+        str.Replace(L"<font color=", L"{c}");
+        str.Replace(L"</i>", L"{\\i}");

```

Comment:
@adipose: Shouldn't there be </font> instead of </i>?

Reply:
Actually these lines should be removed. I forgot to clean them up.